src/components/Menu/MenuItem.tsx

```diff
@@ -28,9 +30,23 @@ export const useStyles = makeStyles(
       '&:focus': {
         outline: 'none',
       },
+      '&:disabled, &[disabled]': {
+        '&, & *': {
+          color: theme.palette.text.disabled,
+        },
+      },
     },
     icon: {
+      alignSelf: 'flex-start',
       marginRight: theme.spacing(1),
+      minWidth: theme.pxToRem(18),
+    },
+    text: {
+      lineHeight: 1.35,
```

Comment:
Curious - does or could the theme have variables for typical line heights so this doesn't have to be hard-coded here? Guessing it would be ideal for this to change along with any changing font-sizes.

Reply:
Adding basic line height standards seems reasonable to me. Also, fine to do in a separate PR.